### PR TITLE
PHP 8.2 support + Laravel 11 support

### LINKED ISSUE #34: php 8.2 support
PHP 8.1 will lose active support on the 25 of November 2023, and security support in 25 Nov 2024

### PR DESCRIPTION
**Pull request by bkintanar**
fix #34

### CODE REVIEW

**Comment by bkintanar** [comment]
@foresight-developers can we add GitHub actions for PHP8.2 please?

**Comment by BilalAslim** [comment]
@bkintanar in the file run_tests.yml line 22 add 8.2 there and it should be added

**Comment by BilalAslim** [comment]
@bkintanar I will look to get this merged in this week.

**Comment by BilalAslim** [comment]
@bkintanar GitHub actions aren't being triggered for some reason, maybe in the work flow file change on to on: [push, pull_request]

https://futurestud.io/tutorials/github-actions-run-on-pull-request#:~:text=Search%20the%20line%20on%3A%20%5Bpush,latest%20steps%3A%20%2D%20name%3A%20%E2%80%A6

let me know if you don't have the time and I can take a look at it in a few weeks time

**Comment by bkintanar-eqho** [comment]
@BilalAslim I think the one in the workflow file is correct. I read this in the link you sent me.

CleanShot 2024-02-21 at 05 06 07@2x

**Comment by BilalAslim** [comment]
@bkintanar ive renabled and ran the workflow on master, can you close this PR and reopen it, maybe it triggers it now. It turned off because 60 days inactivity on the repo. or merge in master maybe the new commit triggers it

**Comment by bkintanar** [comment]
@BilalAslim all green.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+[![Laravel 5.8](https://img.shields.io/badge/Laravel-5.8-informational)](https://github.com/TopOnePercent/baum) [![Laravel 6](https://img.shields.io/badge/Laravel-6-informational)](https://github.com/TopOnePercent/baum) [![Laravel 7](https://img.shields.io/badge/Laravel-7-informational)](https://github.com/TopOnePercent/baum) [![Laravel 8](https://img.shields.io/badge/Laravel-8-informational)](https://github.com/TopOnePercent/baum) [![Laravel 9](https://img.shields.io/badge/Laravel-9-informational)](https://github.com/TopOnePercent/baum) [![Laravel 10](https://img.shields.io/badge/Laravel-10-informational)](https://github.com/TopOnePercent/baum) [![Laravel 10](https://img.shields.io/badge/Laravel-11-informational)](https://github.com/TopOnePercent/baum)
```
Comment (BilalAslim) [inline README.md]:
```suggestion
[![Laravel 5.8](https://img.shields.io/badge/Laravel-5.8-informational)](https://github.com/TopOnePercent/baum) [![Laravel 6](https://img.shields.io/badge/Laravel-6-informational)](https://github.com/TopOnePercent/baum) [![Laravel 7](https://img.shields.io/badge/Laravel-7-informational)](https://github.com/TopOnePercent/baum) [![Laravel 8](https://img.shields.io/badge/Laravel-8-informational)](https://github.com/TopOnePercent/baum) [![Laravel 9](https://img.shields.io/badge/Laravel-9-informational)](https://github.com/TopOnePercent/baum) [![Laravel 10](https://img.shields.io/badge/Laravel-10-informational)](https://github.com/TopOnePercent/baum) [![Laravel 11](https://img.shields.io/badge/Laravel-11-informational)](https://github.com/TopOnePercent/baum)
```

laravel 10 was set twice

**Review by BilalAslim — COMMENTED** [review]
See suggestion

**Comment by BilalAslim** [comment]
@bkintanar Thanks for the help on this, I am now creating a new release once tests have passed on master

**Pull request merged**